scripts/asm-installer/install_asm

```diff
@@ -1693,6 +1730,9 @@ configure_package() {
     run kpt cfg set asm gcloud.project.environProjectID "${ENVIRON_PROJECT_ID}"
     run kpt cfg set asm anthos.servicemesh.hubMembershipID "${HUB_MEMBERSHIP_ID}"
   fi
+  if [[ -n "${CA_NAME}" ]]; then
+    run kpt cfg set asm anthos.servicemesh.external_ca.ca_name "${CA_NAME}"
+  fi
```

Comment:
Unless we have plans for other CAs using the same setter, I'd rather gate this on CA == private_ca as well.